### ISSUE TITLE
Documentation Request: Aliasing

### ISSUE THREAD

**Issue by b-meson** [micahflee/onionshare#680]
I recently was shared an Onionshare link and got to thinking that maybe there is a way to write an alias combining `onionshare`, `torify` and `wget`. since Stem serves up a static "/download/" endpoint maybe we can probably trigger a download by hitting that endpoint. 

Here is what I tried `torify wget -c "http://<endpoint>.onion/<slug>/download"` but it failed 
```
ERROR torsocks[630]: Host unreachable (in socks5_recv_connect_reply() at socks5.c:545)
failed: No route to host.
```

Or maybe I am just complicating things too much what do you think @micahflee @mig5 ?

**Comment by mig5** [comment]
Hi @b-meson, that should work just fine - in fact, it works perfectly fine for me:

<pre>
miguel@clou:~$ torify wget http://rjfjx4oykldtp274.onion/extending-shallot/download
--2018-04-16 11:31:09--  http://rjfjx4oykldtp274.onion/extending-shallot/download
Resolving rjfjx4oykldtp274.onion (rjfjx4oykldtp274.onion)... 127.42.42.0
Connecting to rjfjx4oykldtp274.onion (rjfjx4oykldtp274.onion)|127.42.42.0|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 3014 (2.9K) [application/zip]
Saving to: ‘download’

download                                          100%[=============================================================================================================>]   2.94K  6.28KB/s   in 0.5s   

2018-04-16 11:31:13 (6.28 KB/s) - ‘download’ saved [3014/3014]

</pre>

You are correct that browsing to the /download route invokes the download of the zip directly.

Could the issue have been:

1) your Tor client connection somehow (network, permissions to use torify as your login etc)
2) a momentary issue in negotiating a route to the .onion address via Tor network within a valid length of time (certainly not uncommon)
3) something else I'm not thinking of :)

In short it should work, can you reliably reproduce the issue?

More broadly, I'm not sure on the usefulness of 'aliasing' given that the Onion address/slug is going to be different every time (unless persistent mode is set), you would have to update your alias each time no? (Not sure if by 'alias' you are talking about a bash alias or what)

**Comment by micahflee** [comment]
I didn't just reproduce it, but I've totally used `torify wget` to download files from OnionShare before. Maybe you had an issue with Tor when you tried it. Want to just give it another try @b-meson?
